### ISSUE TITLE
Starting 2nd editor in 2nd project silently fails, opening first browser to first project

### ISSUE THREAD

**Issue by michaelgundlach** [PhaserEditor2D/PhaserEditor2D-v3#179]
## Version

* Phaser Editor 2D Version: 3.31.2

* Operating System: Windows 10 WSL

* Web Browser and version: Chrome 96

## Description

I unzipped Flying_dragon.zip into `~/code/Flying\ dragon`, then did
```cd ~/code/PhaserEditor2D-v3/source/editor
npx phasereditor2d-launcher -editor . -disable-plugins phasereditor2d.code -dev -project ~/code/Flying\ dragon/
```

which **opened an editor to `~/code/breakout` instead!**  Here was the output:

```michael@Swordfish ~/code/Flying dragon
$ npx phasereditor2d-launcher -editor . -disable-plugins phasereditor2d.code -dev -project ~/code/Flying\ dragon/
Launching /home/michael/.phasereditor2d/installs/PhaserEditor2D-core-3.31.2-linux/PhaserEditor2D/PhaserEditor2D ["-editor",".","-disable-plugins","phasereditor2d.code","-dev","-project","/home/michael/code/Flying dragon/"]

2022/01/06 21:10:18 Scanning user home flags: /home/michael/.phasereditor2d/flags.txt

Phaser Editor 2D - v3.31.2

=> Running in Free mode (only 70 files per project allowed)
=> Purchase a license: https://gum.co/phasereditor

2022/01/06 21:10:18 User plugins: /home/michael/.phasereditor2d/plugins
2022/01/06 21:10:18 Loading editor client code at /home/michael/code/Flying dragon
2022/01/06 21:10:18 Loading project at /home/michael/code/Flying dragon
2022/01/06 21:10:18 Development mode enabled
2022/01/06 21:10:18 Listening at http://127.0.0.1:1959/editor

Play IDE online : https://play.phasereditor2d.com/
Documentation   : https://help.phasereditor2d.com/v3
File bugs/ideas : https://github.com/PhaserEditor2D/PhaserEditor2D-v3/
Keep in contact : https://twitter.com/PhaserEditor2D

=> Open the web browser at http://127.0.0.1:1959/editor

2022/01/06 21:10:19 No updates available.
2022/01/06 21:10:19 Opening browser at http://127.0.0.1:1959/editor
```

I found that I had an orphaned process running PhaserEditor2D on `~/code/breakout`.  After killing that process, I repeated the above and it opened the correct project.

Expected behavior: crash with error that port is already in use, possibly by which pid
Actual behavior: claim it all works, open browser to existing server

## Possible solution

This is easy to work around now that I know what the cause was, but it may bite other people new to the editor.

**Comment by PhaserEditor2D** [comment]
It should print something like:

```
2022/01/07 08:21:40 listen tcp :1959: bind: address already in use
```

**Comment by PhaserEditor2D** [comment]
Ok, I tested with npx and it shows the same message:

```
2022/01/07 08:49:52 listen tcp :1959: bind: address already in use
```

I cannot reproduce your behavior.

By the way, this part of the log:

> 2022/01/06 21:10:18 Loading editor client code at /home/michael/code/Flying dragon

It says it is loading the editor sources from the Flying dragon folder?

**Comment by michaelgundlach** [comment]
Right, I keep failing to be in PhaserEditor2d-v3/source/editor when running the npx command.  But I'll get a blank browser when I try that normally, which reminds me to go to the right place.  I see the same behavior even when starting from the right directories.  Here's what I just ran, left console first then right console, and the right console opened breakout instead of Dragon demo (I renamed 'Flying dragon' to 'Dragon demo' to make it match your old screenshots more closely.)

![image](https://user-images.githubusercontent.com/5141333/148558696-dd2b6b6c-3c6a-4079-a812-acb4bd61b04e.png)

I'll close this Issue as I'm probably the only one running into this.  At this point I think I'll stick to writing Phaser code until I'm better at npm and TypeScript :)
